spec/order_spec.rb

```diff
@@ -48,15 +48,18 @@
       it 'should call the SMS class' do
         subject.add(1)
         subject.checkout
-        expect(subject).to respond_to(subject.complete_order).with(SMS.new.send_sms(ENV['MY_PHONE']))
+        expect(subject).to respond_to(:complete_order).with(1).argument
       end
     end
 
```

Comment:
how to actually check that complete_order((ENV['MY_PHONE'])) calls SMS.new.send_sms?